content/blog/tdd-2024/index.Rmd

```diff
@@ -0,0 +1,50 @@
+---
+output: hugodown::hugo_document
+
+slug: tdd-2024
+title: Tidyverse developer data 2024
+date: 2024-04-04
+author: Hadley Wickham
+description: >
+    Join the tidyverse team on August 15 
+
+photo:
+  author: JD Long
+  url: https://cerebralmastication.com
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [tidyverse-dev-day]
+---
+
+```{=html}
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+```
+It's been a hot minute since the last one, but we are very excited announce the next tidyverse developer day will after [posit::conf](https://posit.co/conference/) in Seattle on August 15, 2024!
```

Comment:
```suggestion
It's been a hot minute since the last one, but we are very excited to announce that the next tidyverse developer day will be after [posit::conf](https://posit.co/conference/) in Seattle on August 15, 2024!
```